examples/resources/adfs-connector.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 spec:
   # display allows to set the caption of the "login" button
   # in the Web interface
-  display: "Login with SSO"
+  display: "Login with Active Directory"
```

Comment:
These automatically have "Login with" added to the beginning when shown in the web interface, so we shouldn't put it here or you end up with "Login with Login with Active Directory"